homeassistant/components/cloud/tts.py

```diff
@@ -112,17 +126,20 @@ def async_get_supported_voices(self, language: str) -> list[Voice] | None:
         return [Voice(voice, voice) for voice in voices]
 
     @property
-    def default_options(self):
+    def default_options(self) -> dict[str, Any]:
         """Return a dict include default options."""
         return {
             ATTR_GENDER: self._gender,
             ATTR_AUDIO_OUTPUT: AudioOutput.MP3,
         }
 
-    async def async_get_tts_audio(self, message, language, options=None):
+    async def async_get_tts_audio(
+        self, message: str, language: str, options: dict[str, Any] | None = None
+    ) -> TtsAudioType:
         """Load TTS from NabuCasa Cloud."""
         # Process TTS
         try:
+            assert options is not None
```

Comment:
If this can not be None, this should change instead `options: dict[str, Any] | None = None`

Reply:
I'm not sure, `tts.Provider.async_get_tts_audio` allows options to be `None`, but the cloud implementation did not account for this.
@MartinHjelmare can you confirm which is correct?

Reply:
We can't change it because that would mean changing the signature of `tts.Provider.async_get_tts_audio` in a subclass which is not allowed.

We could adjust `tts.Provider.async_get_tts_audio` to return an empty dict instead of `None` though in a follow-up PR?